tests/providers/google/cloud/hooks/test_bigquery.py

```diff
@@ -946,6 +947,16 @@ def test_dbapi_get_sqlalchemy_engine(self):
         ):
             self.hook.get_sqlalchemy_engine()
 
+    @mock.patch(
+        'airflow.providers.google.common.hooks.base_google.GoogleBaseHook._get_credentials_and_project_id',
+        return_value=(CREDENTIALS, PROJECT_ID),
+    )
+    def test_dbapi_get_sqlalchemy_engine_success(
+        self, mock_insert, mock_get_service, mock_get_creds_and_proj_id
+    ):
+        bq_hook = BigQueryHook(use_legacy_sql=False)
+        assert isinstance(bq_hook.get_sqlalchemy_engine(), sqlalchemy.engine.base.Engine) is True
```

Comment:
added a test case for it

Reply:
don't know how to set `extra__google_cloud_platform__key_path` into connection for testing

remove this unit test for now